src/github3/models.py

```diff
@@ -54,9 +54,8 @@ def _update_attributes(self, json):
 
     def __getattr__(self, attribute):
         """Proxy access to stored JSON."""
-        if attribute == '_json_data':
-            raise AttributeError(attribute)
-        if attribute not in self._json_data:
+        json_data = object.__getattr__(self, '_json_data')
+        if attribute not in json_data:
             raise AttributeError(attribute)
         value = self._json_data.get(attribute)
```

Comment:
This should be updated to use `json_data` too, e.g.,

```suggestion
value = json_data[attribute]
```

Especially since we already verify that `attribute` is a key in `json_data` before this.